### ISSUE TITLE
Make it to where people can upload files to boxed wine even in browser.

### ISSUE THREAD

**Issue by blind09** [danoon2/Boxedwine#39]

**Comment by kevodwyer** [comment]
You are able to manually add files to the root filesystem (before starting boxedwine)
on Windows look at:
 Users/YOUR_NAME/AppData/Roaming/Boxedwine/Containers/APP_CONTAINER_NAME/root/home/username/.wine/drive_c
Emscripten
Add upload/download button by editing boxedwine-shell.js and set:
DEFAULT_AUTO_RUN = false;
Config.showUploadDownload = true;
If you want to persist files (d:/) you can change the filesystem from MEMORY to LOCAL STORAGE via:
Config.storageMode = STORAGE_LOCAL_STORAGE;
There is also an option to use a dropbox account as a back-end, but I haven't tested that in years and probably is now out-of-date.
See [instructions](https://github.com/danoon2/Boxedwine/blob/master/buildFlags.txt) for more options

**Comment by jarble** [comment]
@kevodwyer Is it possible to load `.exe` files from a URL in the browser, instead of manually adding them to the root file system?
I'd like to do this using the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), for example.

**Comment by kevodwyer** [comment]
Yes, you can.  See buildFlags.txt mentioned above for instructions.
Various options:
- directly add fetch() call to the boxedwine-shell.js
- use existing &app-payload=[base64 string] url parameter
- use drag and drop to copy a zip file onto the canvas or use existing upload button
- for full control - build upon boxedwine-shell-simple.js example in the project/emscripten directory
